appserver/admingui/rest-monitoring-plugin/src/main/resources/restmonitoring/restMonitoring.jsf

```diff
@@ -97,10 +95,7 @@ DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
                 <sun:property id="enabledProp"  labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" label="$resource{i18nrest.monitoring.rest.Enabled}" helpText="$resource{i18nrest.monitoring.rest.EnabledHelp}">
                     <sun:checkbox id="isEnabled" label="$resource{i18n.common.Enabled}" selected="#{pageSession.isEnabled}" selectedValue="true"/>
                 </sun:property>
-                <sun:property id="dynamicProp"  labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" label="$resource{i18nrest.monitoring.rest.Dynamic}" helpText="$resource{i18nrest.monitoring.rest.DynamicHelp}">
-                    <sun:checkbox id="isDynamic" label="$resource{i18n.common.Enabled}" selected="#{pageSession.isDynamic}" selectedValue="true"/>
-                </sun:property>
-                <sun:property id="applicationNameProp" labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" label="$resource{i18nrest.monitoring.rest.ApplicationName}" helpText="$resource{i18nrest.monitoring.rest.ApplicationNameHelp}">
+            <sun:property id="applicationNameProp" labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" label="$resource{i18nrest.monitoring.rest.ApplicationName}" helpText="$resource{i18nrest.monitoring.rest.ApplicationNameHelp}">
                     <sun:textField id="name" columns="$int{40}" maxLength="250" text="#{pageSession.valueMap['name']}"/>
```

Comment:
`text="#{pageSession.valueMap['name']}"`

This needs to be _applicationName_, to match: https://github.com/MeroRai/Payara/blob/1458c47841ec1a98ad892cd104f06b654db23517/appserver/payara-appserver-modules/rest-monitoring/rest-monitoring-service/src/main/java/fish/payara/appserver/monitoring/rest/service/admin/GetRestMonitoringConfiguration.java#L118